data/general/11-12.csv

```diff
@@ -0,0 +1,26 @@
+Player,Nation,Pos,Age,MP,Starts,Min,90s,Gls,Ast,G-PK,PK,PKatt,CrdY,CrdR,Gls,Ast,G+A,G-PK,G+A-PK,Matches
+Andrea Pirlo\71c5f16f,it ITA,MF,32,37,37,3315,36.8,3,12,3,0,2,5,0,0.08,0.33,0.41,0.08,0.41,Matches
```

Comment:
Fixed it, and added a script to perform the clean up, which should be run only after first downloading the csv file from fbref.com, because all the pushed files are clean now.